docs/troubleshooting.md

```diff
@@ -34,11 +34,11 @@ Prometheus metrics are exposed locally on port `8888` and path `/metrics`.
 
 For containerized environments it may be desirable to expose this port on a
 public interface instead of just locally. The metrics address can be configured
-by passing the `--metrics-addr` flag to the `otelcol` process. See `--help` for
+by passing the `--set "service.telemetry.metrics.address=<ADDR>` flag to the `otelcol` process. See `--help` for
```

Comment:
Consider to change this as logging, to use the  config yaml as the option.

Reply:
Updated, PTAL